src/cfnlint/custom_rules.py

```diff
@@ -21,29 +23,41 @@
 
-def check(filename, template, rules, runner):
+def check(template, rules, runner):
     """ Process custom rule file """
     matches = []
 
-    with open(filename) as customRules:
-        line_number = 1
-        for line in customRules:
-            LOGGER.debug('Processing Custom Rule Line %d', line_number)
-            rule = cfnlint.customRules.Rule.make_rule(line, line_number)
-            if rule.valid and rule.resourceType[0] != '#':
-                try:
-                    resource_properties = template.get_resource_properties([rule.resourceType])
-                    operator_result = Operator[rule.operator](template, rule, resource_properties)
-                    matches += operator_result
-                except KeyError:
-                    matches.append(cfnlint.rules.Match(
-                        1, 1,
-                        1, 1,
-                        template.filename, cfnlint.customRules.Operators.CustomRule('E9999', 'Error'),
-                        str(rule.operator) + ' not in supported operators: ' + str(list(Operator.keys())) + ' at ' + str(line), None))
-            line_number += 1
-    arg_matches = []
-    for match in matches:
-        if rules.is_rule_enabled(match.rule.id, False):
-            arg_matches.append(match)
-    return runner.check_metadata_directives(arg_matches)
+    if CUSTOM_RULES_FILE is None:
+        return matches
+
+    try:
+        with open(CUSTOM_RULES_FILE) as customRules:
+            line_number = 1
+            for line in customRules:
+                LOGGER.debug('Processing Custom Rule Line %d', line_number)
+                rule = cfnlint.customRules.Rule.make_rule(line, line_number)
+                if rule.valid and rule.resourceType[0] != '#':
+                    try:
+                        resource_properties = template.get_resource_properties([rule.resourceType])
+                        operator_result = Operator[rule.operator](template, rule, resource_properties)
+                        matches += operator_result
+                    except KeyError:
+                        matches.append(cfnlint.rules.Match(
+                            1, 1,
+                            1, 1,
+                            template.filename, cfnlint.customRules.Operators.CustomRule('E9999', 'Error'),
+                            str(rule.operator) + ' not in supported operators: ' + str(list(Operator.keys())) + ' at ' + str(line), None))
+                line_number += 1
+        arg_matches = []
+        for match in matches:
+            if rules.is_rule_enabled(match.rule.id, False):
+                arg_matches.append(match)
+        return runner.check_metadata_directives(arg_matches)
+
+    except EnvironmentError:
+        LOGGER.error('Could not find custom rule file: %s', str(CUSTOM_RULES_FILE))
+    return matches
+
+def set_filename(filename):
+    global CUSTOM_RULES_FILE
+    CUSTOM_RULES_FILE = filename
```

Comment:
https://github.com/aws-cloudformation/cfn-python-lint/pull/1665 should fix the rest but [`Final newline missing`](https://travis-ci.com/github/aws-cloudformation/cfn-python-lint/jobs/377678928#L321) here should be fixed

Reply:
Newline missing has been fixed, amended the latest commit and pushed!